aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -332,14 +333,25 @@ module std::features {
         is_enabled(BN254_STRUCTURES)
     }
 
+    /// If enabled:
+    /// - Buffer on-chain config changes.
+    /// - At the end of every epoch, lock on-chain configs, perform a DKG, unlock on-chain configs, and enter the new epoch.
+    ///
+    /// Lifetime: transient
+    const RECONFIGURE_WITH_DKG: u64 = 45;
+    public fun get_reconfigure_with_dkg_feature(): u64 { RECONFIGURE_WITH_DKG }
+    public fun reconfigure_with_dkg_enabled(): bool acquires Features {
+        is_enabled(RECONFIGURE_WITH_DKG)
+    }
+
     // ============================================================================================
     // Feature Flag Implementation
 
     /// The provided signer has not a framework address.
     const EFRAMEWORK_SIGNER_NEEDED: u64 = 1;
 
     /// The enabled features, represented by a bitset stored on chain.
-    struct Features has key {
+    struct Features has copy, drop, key, store {
```

Comment:
This shouldn't need drop ability. Can be destructed let Features { features } = features;

Reply:
fixed